homebridge/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '2'
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_HOMEBRIDGE_WEB_IP
+      APP_PORT: $APP_HOMEBRIDGE_WEB_PORT
+  homebridge:
+    image: oznu/homebridge:ubuntu
+    container_name: homebridge
+    restart: always
```

Comment:
We currently rely on the `on-failure` restart policy. ie only restarting if the container unexpectedly stops as opposed to always restarting.

Reply:
sounds good!